docs/changelog.txt

```diff
@@ -47,6 +47,7 @@ changelog.txt uses a syntax similar to RST, with a few special sequences:
 ## Documentation
 
 ## API
+- `word_wrap`: add ``trim_whitespace_after_newline`` option, which trims any whitespace at the beginning of a line after wrapping
```

Comment:
This causes ``docs/_auto/news.rst:29: WARNING: undefined label: word_wrap``

Reply:
yeah, it's not a document label. I'd suggest double backticks.